target/linux/generic/hack-5.4/652-netfilter-offload-check-dst.patch

```diff
@@ -0,0 +1,81 @@
+Subject: kernel: check dst of flow offload
+
+Signed-off-by: Ritaro Takenaka <ritarot634@gmail.com>
```

Comment:
Will it be possible if you would add the patch in the same shape at least how did you do in other pull request? At least, there should be proper commit subject, commit description.

What I mean is that there should be this:
```
From 88b937673b3552d54da20f648e61a123f4c1fa67 Mon Sep 17 00:00:00 2001
From: Ritaro Takenaka <ritarot634@gmail.com>
Date: Tue, 17 May 2022 12:55:30 +0200
Subject: netfilter: flowtable: move dst_check to packet path

[ Upstream commit 2738d9d963bd1f06d5114c2b4fa5771a95703991 ]

Fixes sporadic IPv6 packet loss when flow offloading is enabled.

IPv6 route GC and flowtable GC are not synchronized.
When dst_cache becomes stale and a packet passes through the flow before
the flowtable GC teardowns it, the packet can be dropped.
So, it is necessary to check dst every time in packet path.

Fixes: 227e1e4d0d6c ("netfilter: nf_flowtable: skip device lookup from interface index")
Signed-off-by: Ritaro Takenaka <ritarot634@gmail.com>
Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
Signed-off-by: Sasha Levin <sashal@kernel.org>
```

Copy&pasted from: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/net/netfilter?h=v5.10.118&id=88b937673b3552d54da20f648e61a123f4c1fa67

This helps to know what the patch does, who is the author and why it is needed at all.

Reply:
I added a patch description and moved it to backport-5.4. Thanks.